### PR TITLE
Product create ui

### PR DESCRIPTION
**Pull request by austinkwok1997**

### CODE REVIEW

**Comment by austinkwok1997** [comment]
@bonnieli

File: client/src/pages/CreateProduct.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                  <Paper elevation={3} />
+                </div>
+                <div className={classes.rootPaper}>
+                  <Paper elevation={3} />
+                  <Paper elevation={3} />
+                  <Paper elevation={3} />
+                </div>
+              </Grid>
+              <Grid item xs={6}>
+                <form className={classes.rootForm + " margin-top-140"}>
```
Comment (bonnieli) [inline client/src/pages/CreateProduct.js]: I would avoid using a mix of App.css and makeStyles, stick with one. so if you need 140px margin on, i would add it to rootForm, or create another attribute to makeStyles

File: client/src/pages/Landing.js [hunk 2]
```diff
@@ -41,29 +41,30 @@ class LandingPage extends Component {
...
-                );
-              }}
-            />
-          </React.Fragment>
-        )}
-        {this.state.step >= 3 && (
-          <Typography>All done! Now go make a pull request!</Typography>
-        )}
-      </div>
+      // <div className={classes.landingContainer}>
```
Comment (bonnieli) [inline client/src/pages/Landing.js]: you can feel free to remove this entire landing page if it's not dependant anymore!

File: client/src/App.js [hunk 3]
```diff
@@ -4,14 +4,18 @@ import { BrowserRouter, Route } from "react-router-dom";
...
 
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
+      <NavBar />
       <BrowserRouter>
         <Route path="/" component={LandingPage} />
+        <Route path="/createproduct/" component={CreateProduct} />
```
Comment (bonnieli) [inline client/src/App.js]: I would name this `createProduct` or `create-product`

**Comment by austinkwok1997** [comment]
Made the changes @bonnieli suggested

**Review by bonnieli — APPROVED** [review]

**Pull request merged**